packages/fxa-auth-server/test/local/sentry.js

```diff
@@ -22,17 +22,55 @@ const sandbox = sinon.createSandbox();
 
 describe('Sentry', () => {
   let server;
+  let sentryCaptureSpy;
+  let scopeContextSpy;
+  let scopeSpy;
 
   beforeEach(() => {
+    // Sentry fakes and spies
```

Comment:
While adding new tests to check that errors with status code 400 were being ignored I observed inconsistent behavior. If I'd target a single test it might fail, but if I'd run them all it would work. I suspect something about the spies was an issue. The two things I did to address this were hoist the repeated spies (and some setup) to the top level before each, and use the sinon sandbox instead of sinon where possible.